src/couch_stats/src/couch_stats_aggregator.erl

```diff
@@ -30,6 +30,10 @@
     terminate/2
 ]).
 
+
+-define(DEFAULT_INTERVAL, 10).
```

Comment:
The default is repeated in couch_stats.erl as well. Is it worth making an API call in couch_stats which gets the interval and is the only place that knows about the default?

```
collect_interval_sec() ->
      config:get_integer("stats", "interval", ?DEFAULT_INTERVAL).
```

Then aggregator can call that  to get the interval for the histogram?